pkg/backingstore/reconciler.go

```diff
@@ -413,16 +413,22 @@ func (r *Reconciler) ReadSystemInfo() error {
 				r.BackingStore.Name, pool, pool.ResourceType,
 			))
 		}
+		gbsize := int64(20)
 		pvPool := r.BackingStore.Spec.PVPool
-		qty := pvPool.VolumeResources.Requests[corev1.ResourceName(corev1.ResourceStorage)]
-		gbsize, _ := qty.AsInt64()
+		if pvPool.VolumeResources != nil {
+			qty := pvPool.VolumeResources.Requests[corev1.ResourceName(corev1.ResourceStorage)]
+			gbsize, _ = qty.AsInt64()
```

Comment:
Can we use `qty.Value()`?
https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource?tab=doc#Quantity.Value